src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss

```diff
@@ -0,0 +1,139 @@
+@import "./../../../../assets/stylesheets/custom";
+@import "./../../../../assets/stylesheets/mixins";
+@import "./../../../../assets/stylesheets/color-variables";
+.detail-container {
+  width: 100%;
+  float: left;
+  .note{
+    color: #cccccc;
```

Comment:
two questions:

1. Can colors be assigned variables?
2. There is not a hex code of #cccccc in [PatternFly](http://www.patternfly.org/styles/color-palette/#_). If this did not come from the visual designs, then it should be changed to match the library.